model/src/main/resources/reference.conf

```diff
@@ -31,19 +31,19 @@ ort {
 
   licenseFilePatterns {
     licenseFilenames = ["license*"]
-    patentFilenames = ["patents"]
+    patentFilenames = [patents]
```

Comment:
Hmm, I'm not sure esp. about this one. Isn't it confusing to omit quotes here, but not above and below?

Reply:
I first thought the same, but then decided it's more consistent to use quotes only if they are required in all places in the file. But I'd be fine with changing this one back as well if you prefer that.

Reply:
No, I think it's fine to keep it as a show-case that wildcard patterns need to be quoted.